template-for-repository/proofs/Makefile.common

```diff
@@ -37,6 +37,7 @@
 #
 #         HARNESS_FILE=<HARNESS_FILE>
 #         HARNESS_ENTRY=<HARNESS_ENTRY>
+#         PROOF_UID=<PROOF_UID>
```

Comment:
Where does this come from?  What is the proof_uid?  Should the set up script prompt for?  I kind of hope not, but the script should set this.

Reply:
This is explained in the template-for-proof Makefile:
```
# This should be a unique identifier for this proof, and will appear on the
# Litani dashboard. It can be human-readable and contain spaces if you wish.
PROOF_UID = <__FUNCTION_NAME__>
```